Resources/translations/messages.en.yml

```diff
@@ -1,5 +1,6 @@
 aligent.webhook.channel.label: Webhook
 aligent.webhook.webhooktransport.entity_label: Webhook Transport
+aligent.asyncevents.failedjob.entity_plural_label: Failed Jobs
```

Comment:
This translation is a duplicate

Reply:
It is weird because without this translation **aligent.asyncevents.failedjob.entity_plural_label** I would get an warning message in the middle of the Failed Jobs grid saying that the translation above didn't exist, so that's why I created it, or are you talking about the **aligent.async.failedjob.entity_plural_label**? I've just noticed that is duplicate as well and I'll sort that out now.

Reply:
Ahh wait, I found three different translations with the same content but I guess they would be referenced in different scenarios right? Should I remove any of them?
- aligent.asyncevents.failedjob.entity_plural_label
- aligent.async.failedjob.entity**.**plural_label
- aligent.async.failedjob.entity**_**plural_label

Reply:
I guess it's because the bundle was renamed from 'async' to 'async events'. You can leave just one and make sure others are not referenced anywhere